packages/atomic/src/components/insight/result-templates/atomic-insight-result-children/atomic-insight-result-children.tsx

```diff
@@ -74,6 +74,10 @@ export class AtomicResultChildren
    * The non-localized copy for an empty result state.
    */
   @Prop() public noResultText = 'no-documents-related';
+  /**
+   * Whether to show a message when no child results are found.
+   */
```

Comment:
I think we should be more specific here and in the description of the property above. We should specify when exactly this label is shown. Does it show up before pressing "show more"? only after? etc.